### ISSUE TITLE
Computing the hessian

### ISSUE THREAD

**Issue by rolocampusano** [liaison/Multinomial-Logistic-Regression#1]
Hi!

I forked your implementation, and I am using to learning PyTorch and git. Thank you! If you take a look to my repo, my modifications are more about the data manipulation than about the method itself as the data that I have is too big to create “session-alter” specific chunks. I also added the data loader so that it can handle more than one session at the time. 

I need to get standard errors for the weights. In structural work, a good way to proxy them is through the hessian of the last loss function evaluated in the optimal level. In my case, I am using the log likelihood, but I haven't been able to get the hessian out of it using autograd. Do you have an idea about how to do it? 

Thanks!
